docs/application/dotnet/guides/security/privacy-related-permissions.md

```diff
@@ -17,6 +17,12 @@ The main features of the `Tizen.Security.PrivacyPrivilegeManager` class include:
 
 For a list of privacy-related privileges, see [Security and API Privileges](../../tutorials/sec-privileges.md).
 
+> [!NOTE]
+>
```

Comment:
Global: No space after [!Note]

Wrong usage 

\> [!NOTE]
\>
\> Example text 

Right usage 

\> [!NOTE]
\> Example text....

For more information on note styling, kindly refer to: 
https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning

Reply:
Done